mtbf_job_runner.py

```diff
@@ -87,11 +87,16 @@ def add_7mobile_action(self):
 
     @action(enabled=False)
     def change_memory(self):
-        #TODO: use native adb/fastboot command to change memory?
+        # This function only work in flame
+        # TODO: use native adb/fastboot command to change memory?
         # Make sure it's in fastboot mode, TODO: leverage all fastboot command in one task function
+        memory = 512 # default set 512
+        if 'MEM' in os.environ:
+            memory = os.environ['MEM']
+        elif self.settings['change_memory']['enabled'] and 'memory' in self.settings['change_memory']:
```

Comment:
So if environ is set, the settings.json setting will be ignored? Is that what you intended?